conf/mesos-site.xml

```diff
@@ -18,107 +18,37 @@
 
 <configuration>
   <property>
-    <name>mesos.hdfs.data.dir</name>
-    <value>/tmp/hdfs/data</value>
-  </property>
-  
-  <property>
-    <name>mesos.hdfs.secondary.data.dir</name>
-    <value>/var/run/hadoop-hdfs</value>
-  </property>
-  
-  <property>
     <name>mesos.hdfs.framework.mnt.path</name>
     <value>/opt/mesosphere</value>
   </property>
-
+  
   <property>
     <name>mesos.hdfs.state.zk</name>
-    <value>localhost:2181</value>
+    <value>master.mesos:2181</value>
   </property>
 
   <property>
     <name>mesos.master.uri</name>
-    <value>zk://localhost:2181/mesos</value>
+    <value>zk://master.mesos:2181/mesos</value>
   </property>
 
   <property>
     <name>mesos.hdfs.zkfc.ha.zookeeper.quorum</name>
-    <value>localhost:2181</value>
-  </property>
-
-  <property>
-    <name>mesos.hdfs.framework.name</name>
-    <value>hdfs</value>
+    <value>master0.mesos:2181</value>
   </property>
 
   <property>
     <name>mesos.hdfs.mesosdns</name>
-    <value>false</value>
+    <value>true</value>
   </property>
 
   <property>
-    <name>mesos.hdfs.mesosdns.domain</name>
-    <value>mesos</value>
+    <name>mesos.hdfs.preshared-configuration</name>
+    <value>true</value>
   </property>
 
   <property>
     <name>mesos.native.library</name>
-    <value>/usr/local/lib/libmesos.so</value>
-  </property>
-
-  <property>
```

Comment:
We should keep the defaults for memory and cpu here as well. If we need to raise the default values for some of these as it makes sense, that's fine, but we should not remove them. This is an important piece that the user would want to configure.